packages/cardhost/app/lib/builder.ts

```diff
@@ -216,7 +216,8 @@ export default class LocalRealm implements Builder {
     assertDocumentDataIsResource(resource);
     let data = resource.attributes;
     let id = resource.id
-      ? this.parseOwnRealmURL(resource.id)!.id
+      ? // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
@ef4 started seeing these too. we should just add this to our eslint-config package